app/code/community/Bolt/Boltpay/Model/Cron.php

```diff
@@ -44,4 +44,44 @@ public function cleanupQuotes() {
 
         $connection->query($sql);
     }
+
+    /**
+     * After a pre-auth pending order has existed for 15 minutes or more, we remove it from the system.
+     * At this point, Bolt should have called the "failed_payment" webhook to do this. We are catching
```

Comment:
let's explicitly mention that this is accounting for case like magento returns 500 but Bolt proceed to auth?

Reply:
done